frontend/src/v5/ui/components/shared/topNavigation/topNavigation.component.tsx

```diff
@@ -16,31 +16,44 @@
  */
 
 import React from 'react';
+import { useParams, useRouteMatch } from 'react-router-dom';
+
+import { ROUTES, getRouteLink } from '@/v5/services/routes/routes';
 import { Container, Link } from './topNavigaton.styles';
 
-const LINKS = [{
-	title: 'Federations',
-	to: '#federations',
-	active: true,
-}, {
-	title: 'Containers',
-	to: '#container',
-}, {
-	title: 'Tasks',
-	to: '#tasks',
-}, {
-	title: 'Users',
-	to: '#users',
-}, {
-	title: 'Settings',
-	to: '#settings',
-	disabled: true,
-}];
+export const TopNavigation = (): JSX.Element => {
+	const { teamspace, project } = useParams();
+	const isProjectSelected = !!project;
+
+	const links = [{
+		title: 'Federations',
```

Comment:
we need to make sure these strings goes through the ML library and get translated.

Reply:
@sanmont3drepo I assume this has been addressed?

Reply:
yup in another file